exchange/gdpr.go

```diff
@@ -21,7 +32,13 @@ func extractGDPR(bidRequest *openrtb2.BidRequest) (gdpr.Signal, error) {
 }
 
 // ExtractConsent will pull the consent string from an openrtb request
-func extractConsent(bidRequest *openrtb2.BidRequest) (consent string, err error) {
+func extractConsent(bidRequest *openrtb2.BidRequest, gpp gpplib.GppContainer) (consent string, err error) {
+	for i, id := range gpp.SectionTypes {
+		if id == gppConstants.SectionTCFEU2 {
+			consent = gpp.Sections[i].GetValue()
+			return
+		}
+	}
```

Comment:
FYI: I noticed this pattern appears a lot. Consider adding a new helper to the GPP library in the iteration, something like:

```
func (g GppContainer) GetSection(c constants.SectionID) (Section, bool) {
..
```

Where the last bool indicates if the section was found.

We could decide to optimize this lookup with a map as an implementation detail of `GetSection`.

Reply:
Agreed